rust/src/lib.rs

```diff
@@ -1416,4 +1571,28 @@ fn parse_network(value: u32) -> Result<Network, failure::Error> {
         1 => Ok(MainNetwork),
         _ => Err(format_err!("Invalid network type: {}. Expected either 0 or 1 for Testnet or Mainnet, respectively.", value))
     }
-}
\ No newline at end of file
+}
+
+
+
+/// Serialization Helpers for AccountPrivKey 
+/// 
+/// 
+use zcash_primitives::consensus;
+use hdwallet::KeyIndex;
+use hdwallet::traits::Serialize;
+fn extended_private_key_from_seed<P: consensus::Parameters>(
```

Comment:
I'm a bit uncomfortable with duplicating this code, instead of just using the UnifiedSpendingKey::from_seed derivation.

Reply:
This can go away if there was a way serialize an AccountPrivKey since there's no API for doing so we need to fall back to ExtendedPrivKey and use that serialize function